luaui/Widgets/gui_gridmenu.lua

```diff
@@ -2528,8 +2550,8 @@ function widget:CommandNotify(cmdID, _, cmdOpts)
 	end
 end
 
--- widget:UnitCommand(unitID, unitDefID, unitTeam, cmdID, cmdTag, cmdParams, cmdOpts)
-function widget:UnitCommand(unitID, _, _, cmdID, _, cmdParams)
+-- widget:UnitCommand(unitID, unitDefID, unitTeam, cmdParams, cmdOpts, cmdID, cmdTag)
```

Comment:
What is the purpose of this comment?

Reply:
It's there to show what was removed using `_` and it was also there before, so I decided to update it (since it was wrong)

Reply:
Just put them in the function parameters and remove the comment, it's fine for engine function signatures to have the unused parameters in them.